docs/source/bag-api.rst

```diff
@@ -71,6 +71,14 @@ Turn Bags into other things
    Bag.to_delayed
    Bag.to_avro
 
+Random
+------
+
+.. autosummary::
+   random.sample
+   random.choices
+
+
```

Comment:
These seem to be drawing on the real random library rather than bag.random. I'm not sure how to get around that and if makes me a little concerned that this is an odd place to add functionality. Maybe `choice` and `sample` should be methods on bag objects instead.  

Also you'll want to include them in the "Other functions" section below.

Reply:
@jrbourbeau maybe you can weight in on how you think this should be implemented?

Reply:
I simply copied dask arrays doc structure: [check here](https://github.com/dask/dask/blob/master/docs/source/array-api.rst#random)
I think we are doing it right :-)

Reply:
Ah thanks for that pointer. In that case I think you are just missing an import in the dask bag __init_.py `from . import random`

Reply:
You can confirm that you are getting the right docstring by building the docs and making sure they match what's written in the .py file.

Reply:
Hey, I spotted the problem, thanks, I'm gonna work on it.

Reply:
Cool! Let me know if you run into issues.

Reply:
I discovered [this bug](https://github.com/dask/dask/issues/6266) who slowed me down a bit.
Now I solved it and I am gonna PR soon.